docs/standard/generics/covariance-and-contravariance.md

```diff
@@ -52,7 +52,7 @@ Covariance and contravariance are terms that refer to the ability to use a more
   
  Covariance and contravariance are collectively referred to as *variance*. A generic type parameter that is not marked covariant or contravariant is referred to as *invariant*. A brief summary of facts about variance in the common language runtime:  
   
-- In the .NET Framework 4, variant type parameters are restricted to generic interface and generic delegate types.  
+- Variant type parameters are restricted to generic interface and generic delegate types.  
```

Comment:
Not related to your PR: This isn't correct in C# 9.  C#9 also also covariant return types on virtual / override functions.